lib/hammer_cli_foreman/defaults.rb

```diff
@@ -0,0 +1,27 @@
+require 'hammer_cli'
+module HammerCLIForeman
+
+  class Defaults < HammerCLI::Defaults
```

Comment:
Do we need to inherit from the main defaults manager class? I don't see much connection here.